src/components/TabNavigator.styles.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/native'
+import { View } from 'react-native'
+import { Text } from 'react-native-paper'
+import theme from '../utils/theme'
+
+const Label = styled(Text)`
+  color: ${(props) => props.color};
+  margin-bottom: 3px;
+`
+
+const CenterView = styled(View)`
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  background: ${theme.colors.background};
+`
+
+export { Label, CenterView }
```

Comment:
@chadfawcett, is this the way you were hoping we'd use the `Component.js`, `Component.styles.js` file structure?

Reply:
Yup! The only change I'd suggest is to do the exports inline:
```js
export const Label = styled(View)...
```
...etc. I only suggest that because that's the way I've been doing it, but happy to switch to this if you prefer.